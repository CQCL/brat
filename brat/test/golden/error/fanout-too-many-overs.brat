f(Vec(Nat, 3)) -> Nat, Nat
f = { [/\] }
