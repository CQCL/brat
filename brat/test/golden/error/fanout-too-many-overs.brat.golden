Error in test/golden/error/fanout-too-many-overs.brat on line 2:
f = { [/\] }
    ^^^^^^^^

  Type error: No unders but overs: (head :: Nat) for [/\]

